browser/ui/views/overlay/brave_video_overlay_window_views.cc

```diff
@@ -341,6 +367,36 @@ void BraveVideoOverlayWindowViews::Hide() {
   VideoOverlayWindowViews::Hide();
 }
 
+int BraveVideoOverlayWindowViews::GetNonClientComponent(
+    const gfx::Point& point) {
+  int res = VideoOverlayWindowViews::GetNonClientComponent(point);
+  if (seeker_ && seeker_->GetWidget() == this) {
```

Comment:
Q) need to check `seeker_->GetWidget() == this` here? Curious it could have false case.